src/store/flow/flow.ts

```diff
@@ -401,6 +401,4 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
 
 export const DEFAULT_MODES = [
   SupportedMode.SMS,
```

Comment:
Hmm, I recognise this wasn't in the AC but wondering if this can also be configurable?

Reply:
Thank you for the idea, I will add it